src/main.rs

```diff
@@ -90,11 +99,25 @@ fn main() {
         _ => unreachable!(),
     };
 
+    let headers: Vec<(String, String)> = matches
+        .values_of("header")
+        .unwrap_or(Default::default())
+        .map(|v| {
+            let m = v.split(":").collect::<Vec<&str>>();
```

Comment:
I just realized that colon is valid in the second half. I think that if we use `splitn` we can make sure it include all off the second half.

https://doc.rust-lang.org/std/string/struct.String.html#method.splitn

Reply:
Good catch.

Reply:
Good catch!